tests/modules/nf-core/parabricks/genotypegvcf/main.nf

```diff
@@ -8,7 +8,7 @@ workflow test_parabricks_genotypegvcf {
     
     input = [
         [ id:'test', single_end:false ], // meta map
-        file(params.test_data['homo_sapiens']['illumina']['test.genome.vcf'], checkIfExists: true)
+        file(params.test_data['homo_sapiens']['illumina']['test_genome_vcf'], checkIfExists: true)
```

Comment:
@Furentsu this was the line that caused all the problems :)

Reply:
Oh, I see! Thank you :) Here the file is named test.genome.vcf though..why is that? https://github.com/nf-core/test-datasets/tree/modules/data/genomics/homo_sapiens/illumina/gvcf